### PR TITLE
Add Hungarian language

### PR DESCRIPTION
**Pull request by kistom4**

### CODE REVIEW

**Comment by Heziode** [comment]
@kistom4 the file is in english. Is a mistake or the translation is not complete ?

**Comment by Heziode** [comment]
I have rebase `patch-2` into this branch (`patch-1`). Think to update your work before push.
Tell me when the translation is completed.

File: locales/hu.json [hunk 1]
```diff
@@ -66,46 +66,46 @@
...
-            "title": "$t({{gb_key}}) level {{fromLevel, number}} → {{level, number}}",
+            "title": "$t({{gb_key}}) szint {{fromLevel, number}} → {{level, number}}",
             "thead": {
-              "th1": "Places",
-              "th2": "Default rewards",
-              "th3": "FP to put by the owner to secure the place",
-              "th4": "Rewards with rate",
-              "th5": "% of investors bonus (arc bonus)",
-              "th6": "Place free"
+              "th1": "Hely",
```
Comment (Heziode) [inline locales/hu.json]: I don't speak Hungarian, here `places` mean **ranks** instead of **location**.

**Comment by Heziode** [comment]
@kistom4, any progress ?

**Comment by kistom4** [comment]
> @kistom4, any progress ?

Finished the core and the GB Investment section. Please upload it!

File: locales/hu.json (hunk 1 above)
Comment (kistom4) [reply locales/hu.json]: Yes. But after upload my files, I will review it.

**Comment by Heziode** [comment]
For Province, name, GB name, etc., you can use [Official FOE wiki in your language](https://hu.wiki.forgeofempires.com/index.php?title=Kezd%C5%91lap).

**Comment by Heziode** [comment]
It will be released later this day.

**Pull request merged**